### PR TITLE
granted: Add version 0.24.0

### LINKED ISSUE #5650: [Request]: granted
### Prerequisites

- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.

### Criteria

- [X] Non-GUI tool
- [X] Reasonably well-known and widely used (e.g. if it's a GitHub project, it should have at least 500 stars and/or 150 forks)
- [X] English interface (or at least English documentation)
- [X] Latest stable version
- [X] Full version (i.e. not a trial version)
- [X] Fairly standard install (e.g. uses a version-specific download URL, no elaborate pre/post install scripts)

### Name

granted

### Description

Granted is a command line interface (CLI) application which simplifies access to cloud roles and allows multiple cloud accounts to be opened in your web browser simultaneously.

### Homepage

https://github.com/common-fate/granted

### Download Link(s)

https://github.com/common-fate/granted/releases

### Some Indication of Popularity/Repute

847 stars
9 watching
82 forks

### PR DESCRIPTION
**Pull request by aviv-julienjehannet**
Closes #5650

- [X] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/granted.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    },
+    "bin": ["assume.ps1", "granted.exe"],
+    "checkver": "github",
+    "autoupdate": {
+        "architecture": {
+            "32bit": {
+                "url": "https://releases.commonfate.io/granted/v$version/granted_$version_windows_i386.zip"
+            },
+            "64bit": {
+                "url": "https://releases.commonfate.io/granted/v$version/granted_$version_windows_amd64.zip"
```
Comment (aliesbelik) [inline bucket/granted.json:25]:
```suggestion
                "url": "https://releases.commonfate.io/granted/v$version/granted_$version_windows_x86_64.zip"
```

Fix autoupdate.

File: bucket/granted.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    "depends": ["aws"],
+    "architecture": {
+        "32bit": {
+            "url": "https://releases.commonfate.io/granted/v0.21.1/granted_0.21.1_windows_i386.zip",
+            "hash": "41cd7a2d3aff65fb38d81de6ba76f72c0a6c773a82d0f11b8b07228f0b5c52f7"
+        },
+        "64bit": {
+            "url": "https://releases.commonfate.io/granted/v0.21.1/granted_0.21.1_windows_x86_64.zip",
+            "hash": "91ce76167734d3a56182bff4f99a2d80ac9165d4a0c642ba95c85c1f0aa3e25f"
+        }
```
Comment (aliesbelik) [inline bucket/granted.json:15]:
```suggestion
        },
        "arm64": {
            "url": "https://releases.commonfate.io/granted/v0.21.1/granted_0.21.1_windows_arm64.zip",
            "hash": "b0078405f26d6eecfa49efc102eb2271da1d56682e89190cd89c72dc32f55f2f"
        }
```

File: bucket/granted.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    "bin": ["assume.ps1", "granted.exe"],
+    "checkver": "github",
+    "autoupdate": {
+        "architecture": {
+            "32bit": {
+                "url": "https://releases.commonfate.io/granted/v$version/granted_$version_windows_i386.zip"
+            },
+            "64bit": {
+                "url": "https://releases.commonfate.io/granted/v$version/granted_$version_windows_amd64.zip"
+            }
```
Comment (aliesbelik) [inline bucket/granted.json:26]:
```suggestion
            },
            "arm64": {
                "url": "https://releases.commonfate.io/granted/v$version/granted_$version_windows_arm64.zip"
            }
```

Add arm64 version as well.

File: bucket/granted.json [hunk 4]
```diff
@@ -0,0 +1,29 @@
+{
+    "version": "0.21.1",
+    "homepage": "https://github.com/common-fate/granted",
+    "description": "Granted is a command line interface (CLI) application which simplifies access to cloud roles and allows multiple cloud accounts to be opened in your web browser simultaneously.",
+    "license": "MIT",
+    "depends": ["aws"],
```
Comment (aliesbelik) [inline bucket/granted.json:6]:
```suggestion
    "depends": "aws",
```

**Review by aliesbelik — CHANGES_REQUESTED** [review]
- Fix autoupdate link.
- Add arm64 version.

**Comment by aviv-julienjehannet** [comment]
Sorry for the low quality of the initial pull-request.
Thank you so much @aliesbelik for your suggestions.

**Review by tech189 — COMMENTED** [review]
Looks all good to me, only issue is I can't update to v0.23.0 because the arm64 download is not working at the moment.

**Comment by aviv-julienjehannet** [comment]
Some news: Common Fate use GoReleaser to publish the builds and they have to upgrade GoReleaser and rerun the release pipeline. They will report back next week for an update.
In doing that, I'll be upgrading to last 0.23.2 version

**Comment by aviv-julienjehannet** [comment]
@tech189 Chris Norman (from Common Fate) confirms the build fix. All should be ok now.

**Review by tech189 — APPROVED** [review]

**Pull request merged**